servers/src/main/java/tachyon/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -41,6 +42,7 @@
 /**
  * This class is a Thrift handler for file system master RPCs invoked by a Tachyon client.
  */
+@ThreadSafe
```

Comment:
same question here, `FileSystemMaster` doesn't seem threadsafe to me, why is this wrapper thread-safe?